testgrid/config.yaml

```diff
@@ -5721,15 +5721,27 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-11
   - name: kubeadm-gce-1.12
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-12
+    alert_options:
+      alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
```

Comment:
Did you consider using the "plus syntax" for these notifications to make it easier to filter? Right now our mailing list doesn't include any auto-generated messages, but now it will. And people are going to want to filter them out if they don't care about these test failures. Sending to `kubernetes-sig-cluster-lifecycle+kubeadm-test-failures@googlegroups.com` would make it easy to filter and set a precedent for other subprojects that want to do something similar.

Reply:
@roberthbailey thanks for the info.
i saw the `+syntax ` in other dashboard configs, but i din't knew what it meant.

cc @timothysc @BenTheElder 
do you think we should append the emails?

Reply:
I would assume they would come FROM an address that could be filtered, or have a SUBJECT that could also be filtered.  But I think @BenTheElder would know more details there.

Reply:
i'm using gmail and the mailing list is labeled already for me. i can even sub-label it for the test specifics based on that first report email subject we receive eventually.

so i don't really need the change, but if others want it. let me know. :+1:

Reply:
The FROM address is likely to be the same for all testgrid emails (so when we set up kubespray / cluster api / etc you won't be able to filter between them) but maybe the SUBJECT will be sufficient. We can leave it this way until it becomes an issue.